### PR TITLE
add a code coverage tool to the test suite

### PR DESCRIPTION
**Pull request by ibawt**
Adding a code coverage thingy so hopefully we won't put more untested stuff in.

re: full test coverage

### CODE REVIEW

**Comment by ibawt** [comment]
accepting bike sheds on the min. coverage value.  It's ~86% now, so I rounded down.

**Comment by fbogsany** [comment]
LGTM, but please add to the SDK side as well.

File: sdk/test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'simplecov'
+SimpleCov.start
+SimpleCov.minimum_coverage 85
+
+require 'opentelemetry'
```
Comment (fbogsany) [inline sdk/test/test_helper.rb]: Should be `opentelemetry/sdk`.

This is going to be a merge conflict with #59 but I'll deal with that.

File: api/test/test_helper.rb [hunk 2]
```diff
@@ -4,5 +4,8 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-require 'opentelemetry'
+require 'simplecov'
+SimpleCov.start
+
+require 'opentelemetry/sdk'
```
Comment (fbogsany) [inline api/test/test_helper.rb]: Should be just `opentelemetry` - this is the API test helper, not the SDK.

File: sdk/test/test_helper.rb (hunk 1 above)
Comment (ibawt) [reply sdk/test/test_helper.rb]: whoops I thought I added that.

File: api/test/test_helper.rb (hunk 2 above)
Comment (ibawt) [reply api/test/test_helper.rb]: oh I see what happened....I should pay more attention to what *path* that file was in :|

**Pull request merged**